packages/t/temporal-polyfill.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "temporal-polyfill",
+  "description": "A lightweight polyfill for Temporal, successor to the JavaScript Date object",
+  "filename": "global.min.js",
+  "keywords": [
+    "Temporal",
+    "polyfill",
+    "date",
+    "time"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fullcalendar/temporal-polyfill"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "temporal-polyfill",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "global.min.js",
+          "global.esm.js",
+          "index.js",
+          "chunk/classApi.js",
+          "chunk/internal.js"
```

```suggestion
          "chunk/*.js"
```
I think we're safe to use `*` here so handle any other future chunks being imported